vcr/cmd/cmd.go

```diff
@@ -37,6 +37,9 @@ func FlagSet() *pflag.FlagSet {
 	defs := vcr.DefaultConfig()
 	flagSet := pflag.NewFlagSet("vcr", pflag.ContinueOnError)
 	flagSet.Bool("vcr.oidc4vci.enabled", defs.OIDC4VCI.Enabled, "Enable issuing and receiving credentials over OIDC4VCI (experimental).")
+	flagSet.Bool("vcr.oidc4vci.url", defs.OIDC4VCI.Enabled, "Base URL for the OIDC4VCI wallet and issuer endpoints (experimental). "+
```

Comment:
Does this url apply to all /n2n endpoints? If not, how does this impact our current prefix-based routing?

Reply:
It doesn't; since it just adds another endpoint (or set of endpoints) under `/n2n`, which should already be properly exposed/secured with mutual TLS certs. So having them under `/n2n` makes more sense than not having  them under it (since it would require everyone to configure additional paths in their proxies).

Reply:
should we put the urls under the `http.alt` config? This is a path based config so it makes sense to add the `/public` and `/n2n` urls there. Might not be the easiest way to access the urls where they are needed though.

Reply:
Out of scope for this issue, documented discussion: https://github.com/nuts-foundation/nuts-node/issues/2112#issuecomment-1571886906